### PR TITLE
duplicate home visit entry fixed

### PR DESCRIPTION
**Pull request by mahmud6390**
duplicate home visit entry fixed.
in previous based on FormSubmissionId  it'll add/update the table but unfortunatlly
everyTime FormSubmissionId was the new one. so everytime it's added one row.

### CODE REVIEW

File: opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java [hunk 1]
```diff
@@ -125,9 +125,9 @@ public void add(HomeVisit homeVisit) {
             if (StringUtils.isBlank(homeVisit.getSyncStatus())) {
                 homeVisit.setSyncStatus(TYPE_Unsynced);
             }
-            if (StringUtils.isBlank(homeVisit.getFormSubmissionId())) {
-                homeVisit.setFormSubmissionId(generateRandomUUIDString());
-            }
+//            if (StringUtils.isBlank(homeVisit.getFormSubmissionId())) {
```
Comment (allan-allay) [inline opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java]: Will this be needed sometime later or in the future?

Comment (mahmud6390) [reply opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java]: will be delete. let me share the issue status is it solve the duplicate issue?

Comment (allan-allay) [reply opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java]: Yes, I've seen the duplicate issue is solved.

Comment (mahmud6390) [reply opensrp-chw/src/main/java/org/smartregister/chw/repository/HomeVisitRepository.java]: cool i removed the commented code. 
please approved the changes then i can merge it in master.

**Pull request merged**